.github/workflows/snapcraftbuild.yml

```diff
@@ -0,0 +1,29 @@
+name: Build and Release Snap
+
+on:
+  push:
+    branches:
+    - master
+
+jobs:
+  build-and-release:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Build
+      id: build
+      uses: snapcore/action-build@v1
+      with:
+        snapcraft-args: --enable-experimental-package-repositories
+    - name: Review
+      uses: diddlesnaps/snapcraft-review-tools-action@v1
+      with:
+        snap: ${{ steps.build.outputs.snap }}
+        isClassic: 'false'
+    - name: Publish playtest to Edge
+      uses: snapcore/action-publish@v1
+      with:
+        store_login: ${{ secrets.STORE_LOGIN }}
```

Comment:
What do we add here? Some token from the snapcraft store? I'm not familiar so I thought I'd ask.

Reply:
Yes. I can generate it and send you the contents on the Discuss.